src/utils/test-utils.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from "react-router-dom";
+import CardList from "../components/CardList/CardList";
+import { rootReducer } from '../store/rootSlice';
+import { authReducer } from '../store/authSlice';
+
+const initCardsData = [
+    {
+        id: "test id",
+        caption: "This is card header",
+        text: "This is card body",
+        checked: false,
+    }
+];
+const INIT_STORE = {
+    root: {
+        cardsData: initCardsData,
+        addingCard: false,
+        readOnly: false,
+    }
+};
+const INIT_UI = (
+    <BrowserRouter>
+        <CardList />
+    </BrowserRouter>
```

Comment:
Why are you testing CardList in Card component unit tests?
You could create and test Card component only